testcontainers/src/runners/async_runner.rs

```diff
@@ -292,6 +296,57 @@ mod tests {
         Ok(())
     }
 
+
+
+    #[tokio::test]
+    async fn async_run_command_should_map_exposed_port_udp_sctp() -> anyhow::Result<()> {
+        let client = Client::lazy_client().await?;
+        let _ = pretty_env_logger::try_init();
+
+        let udp_port = 1000;
+        let sctp_port = 2000;
+
+        let generic_server = GenericImage::new("simple_web_server", "latest")
+            .with_wait_for(WaitFor::message_on_stdout("server is ready"))
+            // Explicitly expose the port, which otherwise would not be available.
+            .with_exposed_port((udp_port, InternetProtocol::Udp))
+            .with_exposed_port((sctp_port, InternetProtocol::Sctp));
+
+        let container = generic_server.start().await?;
+        container.get_host_port_ipv4(udp_port).await?;
```

Comment:
I guess we don't have a proper solution for getting the port here.

Actually the same port can be used for both `UDP` and `TCP` at the same time

Reply:
Related to https://github.com/testcontainers/testcontainers-rs/pull/655/files#r1638337515